### ISSUE TITLE
0.51 does not build on MacOS

### ISSUE THREAD

**Issue by mgruner** [houseabsolute/perl-code-tidyall#65]
```
    #   Failed test 'root_dir'
    #   at t/lib/Test/Code/TidyAll/Basic.pm line 485.
    #   (in Test::Code::TidyAll::Basic->test_cli)
    #                   ' {backup_ttl => '15m',plugins => {},root_dir => '/private/tmp/Code-TidyAll-ZJZW',verbose => 1}'
    #     doesn't match '(?^:root_dir\ \=\>\ \'\/tmp\/Code\-TidyAll\-ZJZW\')'

```

The issue is probably that /tmp is a symlink to /private/tmp. This used to work in older versions (I currently have 0.46 installed).

**Comment by autarch** [comment]
Current failures with the osx-fix branch - https://gist.github.com/karenetheridge/bbe452c780919b5e5b84ed5798a956f2

**Comment by adamherzog** [comment]
0.52 failed for me for the same reasons. It appears to already be fixed in master, though.

**Comment by autarch** [comment]
Fixed in 0.53 (I think)
